src/Umbraco.Core/Services/MemberService.cs

```diff
@@ -389,16 +389,27 @@ public IEnumerable<IMember> GetAll(long pageIndex, int pageSize, out long totalR
         }
 
         /// <summary>
-        /// Get an <see cref="IMember"/> by email
+        /// Get an <see cref="IMember"/> by email. If RequireUniqueEmailForMembers is set to false, then the first member found with the specified email will be returned.
         /// </summary>
         /// <param name="email">Email to use for retrieval</param>
         /// <returns><see cref="IMember"/></returns>
         public IMember? GetByEmail(string email)
+        {
+            var members = this.GetMembersByEmail(email);
+            return members?.FirstOrDefault();
+        }
```

Comment:
This can be simplified to a single line expression:
```suggestion
            => GetMembersByEmail(email).FirstOrDefault();
```

The CMS codebase also doesn't use the `this` qualifier and since `GetMembersByEmail()` doesn't return null, shouldn't require the null-coalescing operator...